keyboards/hhkb/keymaps/schaeferdev/config.h

```diff
@@ -0,0 +1,19 @@
+// Based off of this section:
+// https://github.com/qmk/qmk_firmware/blob/master/doc/BUILD_GUIDE.md#the-configh-file
+#ifndef CONFIG_SCHAEFERDEV_H
+#define CONFIG_SCHAEFERDEV_H
+
+// Bring in original defaults
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.